apollo-router/build/studio.rs

```diff
@@ -24,6 +22,8 @@ pub fn main() -> Result<(), Box<dyn Error>> {
     content = content.replace("[(js_preEncoded)=true]", "");
     std::fs::write(&reports_out, &content)?;
 
+    println!("cargo:rerun-if-changed={}", reports_src.to_str().unwrap());
```

Comment:
why move this?

Reply:
When I was first investigating the rebuild, it was unclear to me with it further up, I ended up with it at the bottom of the build right before handing off to `tonic_build`.

Worth noting, the location of this line does not affect the cargo's change detection.